Sources/Fullscreen/NotificationCenter/PushNotificationDetailsView.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  Copyright © 2020 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public enum PushNotificationDetails {
+    case link(text: String, title: String, showMagnifyingGlass: Bool)
```

Comment:
Maybe `showMagnifyingGlass` could be something unrelated to the current icon, I guess `showSearchIcon` might work?